app/views/users/show.html.erb

```diff
@@ -1,5 +1,49 @@
 <section class="user-section">
-  <h2><%= "Name: #{@user.name}" %></h2>
+  <%= flash[:message] %>
+
+<%= "#{@show_user.name.upcase }'s PROFILE'" %>
+
+<% if current_user == @show_user %>
+
+  <h1>Send A Friend Request</h1>
+
+  <%= form_for :friendship, url: friendships_path do |f|%>
+    <%= f.select :confirmer_id, 
+      User.all.collect { |u| [ u.name, u.id ] if u.id != current_user.id }.select(&:present?), 
+      {:prompt => 'Select Person'}, class: "select_friend"
+    %>
+
+    <%= f.submit "Send Friend Request" %>
+  <% end %>
+
+
+  <h1>Friend Requests From Others</h1>
+  <ul>
+    <% @f.each do |x| %>
+      <li>
+        <% if x.status == false %>
+          <p><%= x.requester.name %></p>
+          <div class='btn-div'>
+            <%= form_for(:friendships, url: friendship_path(@i), method: :patch ) do |f| %>
+              <%= f.hidden_field :status %>
+              <div>
+                <%= f.submit 'Accept' %> 
+              </div>
+            <% end %>
+            <%= form_for(:friendships, url: friendship_path(@i), method: :delete ) do |f| %>
+              <%= f.hidden_field :status %>
+
+              <div>
+                <%= f.submit 'Reject', class: 'btn-right'%>
+              </div>
+            <% end %>
+          </div>
+        <% end %>
+      </li>
+    <% end %>
+  </ul>
+<% end %>
+
   <article class="timeline">
     <h3>Recent posts:</h3>
     <ul class="posts">
```

Comment:
Remember a user can't see other user's posts until they are friends. Please take care of that here.